pkg/resourcedefinitions/helper.go

```diff
@@ -261,6 +265,15 @@ func alignSchemaRef(nb map[string]any, key string, lsr, rsr *openapi3.SchemaRef)
 				if period+1 < len(d) {
 					lv.Description = string(d[:period+1])
 				}
+
+				switch {
+				case lv.ReadOnly && lv.WriteOnly:
+					lv.Description += " (read-only, write-only)"
```

Comment:
What does read-only + write-only mean?

Reply:
that is what we should point out, if the res def gets this, the maintainer should watch out for this.

Reply:
replace with (restricted) or something else?